cmd/tealdbg/localLedger.go

```diff
@@ -53,6 +69,25 @@ func makeAppLedger(
 	apps := []basics.AppIndex{appIdx}
 	apps = append(apps, txn.Txn.ForeignApps...)
 
+	// first populate balances from the indexer
+	for _, acc := range accounts {
+		fmt.Println("debug hello")
+		if _, ok := balances[acc]; !ok {
+			queryString := fmt.Sprintf("%s/v2/accounts/%s?round=%v", indexerURL, acc, round)
+			fmt.Println(queryString)
+			if resp, err := http.Get(queryString); err == nil {
+				if responseData, err := ioutil.ReadAll(resp.Body); err == nil {
+					var accountResp AccountResponse
+					json.Unmarshal(responseData, &accountResp)
+					balances[acc], err = v2.AccountToAccountData(&accountResp.Account)
+					fmt.Println(balances[acc])
+				}
+			} else {
+				fmt.Println(err)
```

Comment:
I know this is for debugging purposes, but you'll eventually want to return the error so your caller can handle it.
```
resp, err := expr
if err != nil {
  return err
}
etc
```
is one pattern that's used a lot.